docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: '3.4'
+
+services:
+    nginx:
+        build:
+            context: docker
+            dockerfile: NginxDockerfile
+        depends_on:
+            - app
+        networks:
+            - frontend
+        volumes:
+            - .:/app
+
+    app:
+        build:
+            context: .
+            target: app-dev
+        networks:
+            - frontend
+        volumes:
+            - .:/app
+
+
+networks:
+    frontend:
```

Comment:
This network is useless, we already have a `default` one. https://docs.docker.com/compose/networking/#configure-the-default-network